themes/startwords/layouts/authors/list.html

```diff
@@ -0,0 +1,52 @@
+{{ define "pagetype" }}authors{{ end }}
+
+{{ define "main" }}
+<main>
+    <div class="grid">
+        <header>
+            <h1>Authors</h1>
+        </header>
+        <div class="text-container">
+            {{ with .Content }}{{ . }}{{ end }}
+
+            <div class="author-list">
+                {{/* Hugo sorts collections by key by default */}}
+                {{ range $id, $info := $.Site.Data.authors }}
+                    <div id="{{ $id }}">
+                    <h2 class="author-name">{{ $info.name }}
+                    {{- with $info.orcid -}}
+                    &nbsp;<a class="orcid" href="https://orcid.org/{{ . }}"><img alt="ORCID" src="{{ relURL "/img/logos/orcid.svg" }}" width="20" height="20" /></a>
+                    {{- end -}}
+                    </h2>
+                    {{- if (or $info.affiliation $info.title) -}}
+                    <p><i>{{ with $info.title }}{{ . }}{{ end }}{{ if (and $info.title $info.affiliation) }}, {{ end }}{{ with $info.affiliation }}{{ . }}{{ end }}</i></p>
+                    {{- end -}}
+                    {{/* find all articles by this author */}}
+                    {{ $articles := where $.Site.AllPages ".Params.Authors" "intersect" (slice $id) }}
+                    {{ range $articles }}
+                    <div class="article">
+                        <p class="article-title"><a href="{{ .RelPermalink }}">{{ if eq .Type "issue" }}Editorial Introduction{{ else }}{{ .Params.Title }}{{ end }}</a></p> {{/* TODO: language indicator */}}
+                        <p class="article-info">{{/* display co-authors, if any */}}
+                        {{ $coauthors := .Params.Authors | complement (slice $id) }}
```

Comment:
The logic here will have to change. As written, this pulls alphanumeric IDs from the article's `authors:` field, but it should pull the Firstname Lastname `name:` field from `authors.yml`. Currently, the author list page will display coauthors beneath an article as, e.g., "with smithjustin, and murraycourtney".

Reply:
Also, is there any way to only use commas in the case of articles with three or more coauthors? Right now, we get "with smithjustin, and caseyjim" for two coauthors. @rlskoeser

Reply:
good catch! I guess the last arg to `delimit` should just be `" and "` instead of `", and "` as I have it now